Engine/Source/ModuleEditor.cpp

```diff
@@ -123,6 +122,20 @@ bool ModuleEditor::CleanUp()
 }
 
 void ModuleEditor::ShowMainMenuBar() {
+	Panel* quadtreeDebug = mPanels[QUADTREEPANEL];
+	Panel* debugPanel = mPanels[DEBUGPANEL];
+	Panel* timerPanel = mPanels[TIMERPANEL];
+
+	Panel* project = mPanels[PROJECTPANEL];
+	Panel* console = mPanels[CONSOLEPANEL];
+	Panel* hierarchy = mPanels[HIERARCHYPANEL];
+	Panel* pause = mPanels[PAUSEPANEL];
+	Panel* scene = mPanels[SCENEPANEL];
+	Panel* inspector = mPanels[INSPECTORPANEL];
+	Panel* lightning = mPanels[LIGHTNINGPANEL];
+
+	Panel* aboutPanel = mPanels[ABOUTPANEL];
+
```

Comment:
This should be as it was before because now this is initialicing this variables when a lot of times it is not neccesary.